src/control/server/superblock.go

```diff
@@ -159,6 +162,22 @@ func (srv *IOServerInstance) CreateSuperblock(msInfo *mgmtInfo) error {
 	return srv.WriteSuperblock()
 }
 
+func (srv *IOServerInstance) logScmStorage() error {
```

Comment:
this is duplicating what storage scan does, so it doesn't make sense to scatter different routines to do the same thing in different areas of code, leads to fragmentation and a code base that's difficult to reason about. this kind of code belongs in server/storage/scm

Reply:
To me, it made the most sense here, because it's only used in the server package. It's not a duplication of storage scan -- it is specifically about reporting on the SCM mountpoint similar to how `df` works. The intent is to log this to the server console on startup to make it easier to spot problems (e.g. SCM wasn't wiped before starting a test run).

Reply:
Given that we need to know the mountpoint (`path.Dir(srv.superblockPath())`), and we need to do string formatting specifically for logging to the server console, what is your proposal for code reorganization?

Reply:
yes duplication was the wrong word, I mean this is a nonintuitive place to store SCM storage -related logic. Instead we could pass path into instance.scmProvider.GetMntSize() which returns total and available bytes then the logic is located in an obvious place for the next person who needs it.

Reply:
Ok, so at that point, GetMntSize() is just a thin wrapper around syscall.Statfs() that returns two float64 values. I'd argue that it still doesn't belong specifically in the scm provider -- it should probably be in `control/provider/system` instead if we're really trying to make it reusable code.

Reply:
I'd also argue that it's still not particularly useful as a reusable library function at that point. If we want to get any other details we'll have to modify it or write another wrapper around statfs. I could understand the objection if this were a 30-line method with complex logic, but it's really statfs doing the work here.